charts/render.py

```diff
@@ -254,6 +254,13 @@ def patch_grafana_dashboards(manifest):
     return manifest
 
 
+def patch_kubelet_service_monitor(manifest):
+    manifest["spec"]["endpoints"][0][
+        "scrapeTimeout"
+    ] = "__var__(prometheus.spec.config.kubelet_scrape_timeout)"
+    return manifest
```

Comment:
Why ? Instead of overloading this in the `charts/kube-prometheus-stack.yaml`:
```
kubelet:
  serviceMonitor:
    scrapeTimeout: "__var__(prometheus.spec.config.kubelet_scrape_timeout)"
```
And let the script handle that for you.

Reply:
👍  for some reason I hadn't found it configurable in the chart last week but in fact it is (https://github.com/scality/metalk8s/blob/development/2.11/charts/kube-prometheus-stack/templates/exporters/kubelet/servicemonitor.yaml#L45) so yes I can change it to what you suggested, good catch !!

Reply:
Done in 9ca67a0442b2eb1dee63f9b5bed84eda288c9e03